README.md

```diff
@@ -15,16 +15,15 @@ PDK includes testing tools, a complete module skeleton, and command line tools t
 
 PDK includes the following tools:
 
-Tool   | Description
-----------------|-------------------------
-metadata-json-lint | Validates and lints `metadata.json` files in modules against  Puppet's module metadatastyle guidelines.
-pdk | Command line tool for generating and testing modules
-puppet-lint | Checks your Puppet code against the recommendations in the Puppet Language style guide.
-puppet-syntax | Checks for correct syntax in Puppet manifests, templates, and Hiera YAML.
-puppetlabs_spec_helper | Provides classes, methods, and Rake tasks to help with spec testing Puppet code.
-rspec-puppet | Tests the behavior of Puppet when it compiles your manifests into a catalog of Puppet resources.
-rspec-puppet-facts | Adds support for running rspec-puppet tests against the facts for your supported operating systems.
-puppet-debugger | Provides a REPL based debugger console.
```

Comment:
@pmcmaw think we need to add puppet-debugger back in for now.

Reply:
@chelnak should facterdb be in there even though it's a distant dependancy it is still significant in what it does

Reply:
Thanks for that @chelnak I missed that and removed by accident, nice catch. I have also included facterdb in both the readme and the markdown docs.

Reply:
Addressed, the diff isn't pleasant to look at as I added puppet-debugger and facterdb in the table in alphabetical order.